Cython/Utility/CythonFunction.c

```diff
@@ -1242,6 +1243,7 @@ static int __pyx_CyFunction_init(PyObject *module) {
     if (unlikely(mstate->__pyx_CyFunctionType == NULL)) {
         return -1;
     }
+    __Pyx_RegisterCommonTypeWithAbc((PyObject*)mstate->__pyx_CyFunctionType, "cython_function_or_method_abc");
```

Comment:
This seems an unusual name for an ABC class. Why not `"CythonFunction"`?
```suggestion
    __Pyx_RegisterCommonTypeWithAbc((PyObject*)mstate->__pyx_CyFunctionType, "CythonFunction");
```